cmdopt/src/main/scala/pravda/cmdopt/CommandLine.scala

```diff
@@ -0,0 +1,216 @@
+package pravda.cmdopt
+
+object CommandLine {
+
+  sealed trait Verb[C, +A]
+
+  final case class Head[C](
+      name: String,
+      text: String = "",
+      desc: String = "",
+      title: String = "",
+      action: (Any, C) => C = (_: Any, c: C) => c,
```

Comment:
What's a purpose of that `action`? Why it's untyped?

Reply:
Oh, it's rudimentary. Removed it. Nice catch!